### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by lucacome**
### Proposed changes
Update docs for 3.2.0 release

### CODE REVIEW

**Review by shaun-nx — APPROVED** [review]

**Review by jjngx — APPROVED** [review]

**Review by ADubhlaoich — APPROVED** [review]
LGTM!

File: docs/content/releases.md [hunk 1]
```diff
@@ -7,6 +7,46 @@ toc: true
...
+27 June 2023
+
+FEATURES:
+
+* [3790](https://github.com/nginxinc/kubernetes-ingress/pull/3790) Gunzip for VS
+* [3863](https://github.com/nginxinc/kubernetes-ingress/pull/3863) OIDC - relaxed OIDC scope validation
+* [3925](https://github.com/nginxinc/kubernetes-ingress/pull/3925) Specify runAsNonRoot in daemon-set manifests. Thanks to [Valters Jansons](https://github.com/sigv).
+* [4001](https://github.com/nginxinc/kubernetes-ingress/pull/4001) Add support for the SameSite sticky cookie attribute.
+* [4031](https://github.com/nginxinc/kubernetes-ingress/pull/4031) Add Alpine image with FIPS inside.
+
```
Comment (brianehlert) [inline docs/content/releases.md:21]: We should add a line about license tracking, even though it is delivered separate.  
The linked PR can be the doc.
"Added utilization tracking for supported (paid) customers"

Comment (brianehlert) [inline docs/content/releases.md:21]: There should be a mention of Google marketplace, even though it is not complete.

File: docs/content/releases.md [hunk 2]
```diff
@@ -7,6 +7,46 @@ toc: true
...
+27 June 2023
+
+FEATURES:
+
+* [3790](https://github.com/nginxinc/kubernetes-ingress/pull/3790) Gunzip for VS
+* [3863](https://github.com/nginxinc/kubernetes-ingress/pull/3863) OIDC - relaxed OIDC scope validation
+* [3925](https://github.com/nginxinc/kubernetes-ingress/pull/3925) Specify runAsNonRoot in daemon-set manifests. Thanks to [Valters Jansons](https://github.com/sigv).
+* [4001](https://github.com/nginxinc/kubernetes-ingress/pull/4001) Add support for the SameSite sticky cookie attribute.
+* [4031](https://github.com/nginxinc/kubernetes-ingress/pull/4031) Add Alpine image with FIPS inside.
```
Comment (brianehlert) [inline docs/content/releases.md:20]: This should mention it is for "supported (paid) customers"

**Review by brianehlert — CHANGES_REQUESTED** [review]

File: docs/content/releases.md [hunk 3]
```diff
@@ -7,6 +7,48 @@ toc: true
...
+FEATURES:
+
+* [3790](https://github.com/nginxinc/kubernetes-ingress/pull/3790) Gunzip for VS
+* [3863](https://github.com/nginxinc/kubernetes-ingress/pull/3863) OIDC - relaxed OIDC scope validation
+* [3925](https://github.com/nginxinc/kubernetes-ingress/pull/3925) Specify runAsNonRoot in daemon-set manifests. Thanks to [Valters Jansons](https://github.com/sigv).
+* [3951](https://github.com/nginxinc/kubernetes-ingress/pull/3951) Add NGINX Plus images to Google Marketplace.
+* [3954](https://github.com/nginxinc/kubernetes-ingress/pull/3954) Add utilization tracking for supported (paid) customers.
+* [4001](https://github.com/nginxinc/kubernetes-ingress/pull/4001) Add support for the SameSite sticky cookie attribute.
+* [4031](https://github.com/nginxinc/kubernetes-ingress/pull/4031) Add NGINX Plus Alpine image with FIPS inside for supported (paid) customers.
+
```
Comment (brianehlert) [inline docs/content/releases.md:23]: We should mention the new doc for how to customize OIDC using config map.

**Pull request merged**